routes/web.php

```diff
@@ -1003,6 +1003,15 @@ function (): void {
         ->middleware('can:delete,experienceSkill')
         ->name('experience-skill.destroy');
 
+    Route::post('experience-skills/batch-store', 'Api\ExperienceSkillsController@batchStore')
+        ->middleware('can:create,App\Models\ExperienceSkill')
+        ->name('experience-skill.batch-store');
+    Route::put('experience-skills/batch-update', 'Api\ExperienceSkillsController@batchUpdate')
+        ->name('experience-skill.batch-update');
+    Route::put('experience-skills/batch-destroy', 'Api\ExperienceSkillsController@batchDestroy')
+        ->name('experience-skill.batch-destroy');
```

Comment:
I think we should make these both POST requests, since they don't really match a REST api design, and don't really reflect the normal PUT use case.